entrypoint.sh

```diff
@@ -1,21 +1,19 @@
 #!/bin/sh -l
 set -e
 
-if [ $# -lt 3 ]
+if [ $# -lt 4 ]
 then
   echo "Some parameters are missing"
   exit 1
 fi
-export KBC_STORAGE_TOKEN=$1
-export COMPONENT_ID=$2
-export TAG=$3
-CONFIGS=$4
 
-docker pull -q quay.io/keboola/syrup-cli:latest
-TASK='
-  echo "Running config {} with tag $TAG ..." &&
-  docker run --rm -e KBC_STORAGE_TOKEN quay.io/keboola/syrup-cli:latest run-job $COMPONENT_ID {} $TAG &&
-  echo "Config {} OK"
-'
-parallel --env COMPONENT_ID --env TAG --lb -j 0 --halt now,fail=1 "$TASK" ::: $CONFIGS
-echo "OK"
+HOST=$1
+TOKEN=$2
+COMPONENT=$3
+TAG=$4
+for id in $5
+do
+  JOBS="${JOBS} $COMPONENT/$id@$TAG"
+done
+
+kbc remote job run --storage-api-host $HOST --storage-api-token $TOKEN $JOBS
```

Comment:
Mohol by si prosim vytvorit testovaci PR v `db-extractor-mysql`, a upravit tam tag na `use-kac-cli`?
https://github.com/keboola/db-extractor-mysql/blob/90979e5d87a3d62bc7a5b9fd2260333433c2f3c8/.github/workflows/push.yml#L85-L86

Iba na vyskusanie, ked CI prejde, tak je to za mna OK.

Reply:
https://github.com/keboola/db-extractor-mysql/actions/runs/4193849916/jobs/7271299970#step:3:21

Reply:
:slightly_frowning_face: no to nam nedoslo, ze tam mozu mat non-master token, ale my zatial vsade vyzadujeme master token

Reply:
https://github.com/keboola/keboola-as-code/blob/3d9e6e0e3e211447a57e037f33f2a61d3d4eb8af/internal/pkg/service/common/dependencies/project.go#L36-L39

Reply:
@jprochazk potrebujes dostat nejaku option `WithoutMasterToken`:
- z cmd https://github.com/keboola/keboola-as-code/blob/3d9e6e0e3e211447a57e037f33f2a61d3d4eb8af/internal/pkg/service/cli/cmd/remote/job/run.go#L43
- az do project dependencies: https://github.com/keboola/keboola-as-code/blob/3d9e6e0e3e211447a57e037f33f2a61d3d4eb8af/internal/pkg/service/common/dependencies/project.go#L36-L39

Potrebujes s tym nejak pomoct, alebo je to jasne? 

Je mi luto, ze je tu este toto prekvapenie.

Reply:
Jo, udělám na to task

Reply:
Už běží https://github.com/keboola/db-extractor-mysql/actions/runs/4193849916/jobs/7329607783